aiter/ops/triton/moe_op_gelu.py

```diff
@@ -43,12 +43,6 @@ def moe_set_quant_func(func):
     _MOE_A_QUANT_FUNC = func
 
 
-@triton.heuristics(
-    {
-        "GRID_MN": lambda args: triton.cdiv(args["EM"], args["BLOCK_SIZE_M"])
-        * triton.cdiv(args["N"], args["BLOCK_SIZE_N"])
-    }
-)
 @triton.jit
 def _fused_moe_kernel(
```

Comment:
Does the EVEN_K optimization not help with this kernel?

Reply:
Ah just missed adding it there